src/webserver/webserver.c

```diff
@@ -716,6 +722,12 @@ void http_init(void)
 
 	mg_set_request_handler(ctx, "/$", redirect_root_handler, NULL);
 
+	if(strcmp(config.webserver.paths.webhome.v.s, "/") == 0 &&
+	   config.dns.blocking.mode.v.blocking_mode == MODE_IP)
+	{
+		log_warn("Webhome is set to root (/) and IP blocking is enabled!");
```

Comment:
```suggestion
		log_warn("Webhome is set to root (/) and IP blocking is enabled. This may result in the Pi-hole web interface to display in places where otherwise ads would show up");
```